public/diagnostic.js

```diff
@@ -23,3 +23,31 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  price: this.price,
+  quantity: this.quantity,
+  orderPrice: Ember.computed('price', 'quantity', function() {
+    return `${this.get('price')}` * `{this.get('quantity')}`;
```

Comment:
I'm not sure if/how this would work with the string interpolation. I'm not confirming it's wrog, I'm just not positive and can't test it at the moment. 

I have this in my solution `return this.get('price') * this.get('quantity');` and that makes more sense to me.